spec/factories/users.rb

```diff
@@ -0,0 +1,12 @@
+FactoryBot.define do
+  factory :user do
+    status { "1" }
+    net_state { Time.now }
+    profile_id { "1" }
+  end
+
+  after(:create) do |user, evaluator|
+    user.profile_id = user.id 
```

Comment:
this is wrong.
why would you do it like that?

Reply:
At the moment I don't have Profile model and its constraints, so I can't reference to nothing. I supposed that on this stage profile_id will be the same as user_id. I added this after action because I'm unable to get user_id in factory_bot: it'll be generated on database level at the moment of creation.